src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -141,4 +162,17 @@ public long getFilteredCount() {
 		return new ResponseEntity<>(this.germplasmService.getGermplasmAttributesByCodes(codes), HttpStatus.OK);
 	}
 
+	@RequestMapping(value = "/crops/{cropName}/germplam-sets/templates/xls", method = RequestMethod.GET)
```

Comment:
Let's use germplasm instead of germplam-sets (notice it is misspelled in your code)